content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,207 @@
+# Application Registration Prerequisites / Developer Guide
+
+## Audience
+
+This technical specification is targeted at software developers building applications on top of Cerner's Ignite application platform as part of the [Cerner CODE ecosystem](https://code.cerner.com/).
+
+## Overview & Purpose
+
+This technical specification outlines a series of security best practices that, if followed, enable Cerner to provide more detailed information about your application to end users. The goal of these best practices is to encourage the use of existing internet security features and standards. In particular, we want users to:
+
+- be assured that their data is being securely transmitted using modern transport layer security and encryption protocols
+- understand what legal entity they're sharing their data with
+
+These best practices should be considered as strongly-recommended prerequisites that software developers address during the application registration process, prior to making their applications available for general use.  Failure to comply with these prerequisites will result in Cerner describing your application as an "Unknown Application" to users during the OAuth authorization workflow.
+
+### Glossary of Terms
+
+* eTLD - Effective Top Level Domain is the base domain. eTLD values are defined in the [Public Suffix List](https://publicsuffix.org/list/public_suffix_list.dat).  Examples include: .com, .edu, .org, and .co.uk.
```

Comment:
Shall we bold each one of these?
**eTLD** -
**eTLD+1** - 
**OV** - 
...

Reply:
+1 They were bolded in the original draft:  https://wiki.cerner.com/pages/viewpage.action?pageId=2663941696

Reply:
I made changes to the files in the pull request and added the bolding.